src/Framework/MockObject/Runtime/Stub/ClosureMock.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Framework\MockObject\Stub;
+
+use PHPUnit\Framework\InvalidArgumentException;
+use PHPUnit\Framework\MockObject\Builder\InvocationMocker;
+use PHPUnit\Framework\MockObject\MethodCannotBeConfiguredException;
+use PHPUnit\Framework\MockObject\MethodNameAlreadyConfiguredException;
+use PHPUnit\Framework\MockObject\Rule\InvocationOrder;
+
+/**
+ * @mixin \PHPUnit\Framework\MockObject\MockObject
+ *
+ * @method InvocationMocker method($constraint)
```

Comment:
I don't understand why Psalm does not recognize this annotation. PhpStorm does, and I used the same annotation as in `MockObject`: https://github.com/sebastianbergmann/phpunit/blob/437af97b42ed0431819f44a45a85f3c6bef8fac9/src/Framework/MockObject/Runtime/Interface/MockObject.php#L16